src/js/components/Grommet/__tests__/Grommet-test.js

```diff
@@ -112,6 +117,29 @@ describe('Grommet', () => {
     }, 600); // wait the aria-live container to clear
   });
 
+  test('custom id announce', done => {
+    const id = 'custom-announcer';
```

Comment:
I don't think we need the CustomAnnouncer test anymore since we aren't taking `id`.